ops/model.py

```diff
@@ -3489,3 +3525,57 @@ def validate_label_value(cls, label: str, value: str):
         if re.search('[,=]', v) is not None:
             raise ModelError(
                 f'metric label values must not contain "," or "=": {label}={value!r}')
+
+
+class LazyNotice:
```

Comment:
I might have said this when we talked about it earlier, but in case not: I really like this approach.